src/components/results.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable indent */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Result extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: props.result,
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    // Typical usage (don't forget to compare props):
+    if (this.props.result !== prevProps.result) {
+      // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
- [ ] Please do not abuse the disabling of linters, instead, try to find a solution for that issue. 😄
    - Linters checks are part of your project to teach you and help you with the best practices. 🚀